.github/workflows/coordinator-test-v21.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         name: Setup Java JDK
         with:
           distribution: 'temurin'
-          java-version: 11
+          java-version: 17
```

Comment:
I would keep this file and ensure that everything keeps working on jdk11 and create a second workload that builds on jdk22